index.js

```diff
@@ -5,13 +5,16 @@ const app = express();
 const sequelize = require("./util/database");
 const role = require("./models/role");
 const user = require("./models/user");
-const accessToken = require("./models/accessToken");
+const Token = require("./models/accessToken");
+const address = require("./models/address");
```

Comment:
try to be consistent about variable cases
const Token = require("./models/accessToken");
const address = require("./models/address");
both are models, but for 1 you used `T`oken for another `a`ddress
both should be same